.gitignore

```diff
@@ -27,3 +27,10 @@ dist-ssr
 # Synpress directories and files
 screenshots
 videos
+
+# Yarn related
+.yarnrc.yml
+.yarn/install-state.gz
```

Comment:
Generally, you should include `.yarnrc.yml` in your Git repo because it contains Yarn configuration specific to your project.

As for `.yarn/install-state.gz`, it is typically related to the installation state of your dependencies. It is generally not necessary to include it in your repository.

Reply:
I don't see these files in our repo. Are these files generated somewhere?

Reply:
They are created when I run the usual commands on my machines.

Reply:
I suggest we avoid adding `.yarnrc.yml` for now, as we might need to create a custom Yarn configuration in the future.